test/images/nonewprivs/Makefile

```diff
@@ -0,0 +1,33 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+.PHONY: all image push clean
+
+TAG = 1.0
+PREFIX = gcr.io/google_containers
```

Comment:
Eh, you can leave it. We're gradually transitioning everything to `google-containers`, but we should probably just send a separate PR to fix all of these.